verification/ci-verify.py

```
@@ -0,0 +1,1548 @@
+
+# ---------------------------------------------------------------------------
+# Global results accumulator
+# ---------------------------------------------------------------------------
+RESULTS: list[tuple[str, bool | None]] = []
+
+
+# ---------------------------------------------------------------------------
+# Core helpers
+# ---------------------------------------------------------------------------
+
+def run_cli(args: list[str], capture_output: bool = True) -> subprocess.CompletedProcess:
```

Ruff (0.15.17)

[warning] Boolean-typed positional argument in function definition

(FBT001)